spec/system/authz_submitter_spec.rb

```diff
@@ -15,6 +15,8 @@
       Collection.create_defaults
       stub_s3
       stub_datacite(host: "api.datacite.org", body: datacite_register_body(prefix: "10.34770"))
+      # Make the screen larger so the save button is alway on screen. This avoids random `Element is not clickable` errors
```

Comment:
@hectorcorrea another thing we may want to ticket.  Should we just make the screen larger to begin with?  I was wondering this myself when I copied this the other week.

Reply:
Ticketed it: https://github.com/pulibrary/pdc_describe/issues/492